bin/index.js

```diff
@@ -106,6 +107,73 @@ const buildConfigurationDescriptor = (manifest) => {
   };
 };
 
+const addExchangeUrl = (manifest) => {
+  return inquirer.prompt([
+    {
+      type: 'input',
+      name: 'exchangeUrl',
+      message: 'Please provide the URL to your extension’s listing on Adobe Exchange. ' +
+        'Leave it blank if you don\'t have one.',
+      validate(input) {
+        if (
+          input.length &&
+          !input.match(
+            /^https:\/\/www\.adobeexchange\.com\/experiencecloud\.details\..+\.html$/gi
+          )
+        ) {
+          return (
+            'Must match the pattern ' +
+            '"https://www.adobeexchange.com/experiencecloud.details.######.html".'
+          );
+        }
+
+        return true;
+      }
+    }
+  ]).then(({ exchangeUrl }) => {
+    if (exchangeUrl) {
+      manifest.exchangeUrl = exchangeUrl;
+    }
+  });
+};
+
+const addIconPath = (manifest) => {
+  return inquirer.prompt([
+    {
+      type: 'input',
+      name: 'iconPath',
+      message: 'The relative path to the icon that will be displayed for the extension within ' +
+        'Launch. It should not not begin with a slash. It must reference an SVG file with a .svg ' +
+        'extension. The SVG should be square and may be scaled by Launch. Leave it blank if you ' +
+        'don\'t have one.',
```

Comment:
It seems like `Leave it blank if you don't have one.` is basically providing a way to cancel out of this prompt. A couple thoughts here:

We don't really support this anywhere else. For example, if I start adding an action type and it asks me about a name and whatnot, I don't have a way to cancel out of that. I'm wondering if we should be consistent?

Saying `Leave it blank if you don't have one` is slightly awkward here, since we also allow the user to enter in a path even if they don't have an icon. Maybe we should say something like `Leave the value blank to cancel.` or even `Hit the ESC key to cancel.` which might seem more natural ([discussed by others here](https://github.com/SBoudrias/Inquirer.js/pull/455)).